src/pages/map/map.ts

```diff
@@ -184,12 +185,16 @@ export class MapPage {
     this.mapMessage = this.translateService.instant('pages.map.geolocation');
 
     return this.geolocation
-      .getCurrentPosition()
+      .getCurrentPosition({ timeout: 5000 })
       .then(result => {
         this.setGeolocationDone();
         return turf.point([ result.coords.longitude, result.coords.latitude ]);
-      }).catch(err => {
-        this.setGeolocationDone(this.translateService.instant('pages.map.geolocationError'));
+      }).catch((err: PositionError) => {
+        if (err.code === PositionErrorValues.TIMEOUT.code) {
```

Comment:
As per the documentation, you should be able to use `PositionError.TIMEOUT` directly, thus having no use for this additional `PositionErrorValues` class.

Reply:
Well, I thought that too, but I don't see how to use those [constants](https://github.com/apache/cordova-plugin-geolocation#constants).

When replacing `PositionErrorValues.TIMEOUT.code` by `PositionError.TIMEOUT`, TS screams at me telling me that I'm using a type (`PositionError`) as a value. When I try to do `err.TIMEOUT`, TS now tells me that ther is not `TIMEOUT` property on a `PositionError` object...